engines/gob/save/saveload_v7.cpp

```diff
@@ -28,58 +28,1328 @@ namespace Gob {
 
 SaveLoad_v7::SaveFile SaveLoad_v7::_saveFiles[] = {
 	// Addy Junior Base
-	{"visage01.inf", kSaveModeSave, nullptr, "face"         }, // Child 01
-	{"visage02.inf", kSaveModeSave, nullptr, "face"         }, // Child 02
-	{"visage03.inf", kSaveModeSave, nullptr, "face"         }, // Child 03
-	{"visage04.inf", kSaveModeSave, nullptr, "face"         }, // Child 04
-	{"visage05.inf", kSaveModeSave, nullptr, "face"         }, // Child 05
-	{"visage06.inf", kSaveModeSave, nullptr, "face"         }, // Child 06
-	{"visage07.inf", kSaveModeSave, nullptr, "face"         }, // Child 07
-	{"visage08.inf", kSaveModeSave, nullptr, "face"         }, // Child 08
-	{"visage09.inf", kSaveModeSave, nullptr, "face"         }, // Child 09
-	{"visage10.inf", kSaveModeSave, nullptr, "face"         }, // Child 10
-	{"visage11.inf", kSaveModeSave, nullptr, "face"         }, // Child 11
-	{"visage12.inf", kSaveModeSave, nullptr, "face"         }, // Child 12
-	{"visage13.inf", kSaveModeSave, nullptr, "face"         }, // Child 13
-	{"visage14.inf", kSaveModeSave, nullptr, "face"         }, // Child 14
-	{"visage15.inf", kSaveModeSave, nullptr, "face"         }, // Child 15
-	{"visage16.inf", kSaveModeSave, nullptr, "face"         }, // Child 16
-	{  "enfant.inf", kSaveModeSave, nullptr, "children"     },
-	{   "debil.tmp", kSaveModeSave, nullptr, nullptr              },
-	{  "config.inf", kSaveModeSave, nullptr, "configuration"},
-// Addy 4 Base
-	{"config00.inf", kSaveModeSave, nullptr, nullptr              },
-	{"statev00.inf", kSaveModeSave, nullptr, nullptr              },
+	{"DATA\\visage01.inf", kSaveModeSave, nullptr, "face"         }, // Child 01
```

Comment:
Why is it adding Windows-style paths?

Reply:
We need to retain the path, not only the filename, as the scripts sometimes request to delete several files matching a path pattern (e.g. "DATA\\?????01.inf"). But this makes me realize this is probably broken on Mac versions of the original game, that would use a different path separator - I will normalize to "/" before doing the lookup.